pinot-perf/src/main/java/org/apache/pinot/perf/BenchmarkDataBlock.java

```diff
@@ -74,22 +77,26 @@ public static void main(String[] args)
         .addProfiler(GCProfiler.class));
   }
 
-  @Param(value = {"INT", "LONG", "STRING", "BYTES", "BIG_DECIMAL", "LONG_ARRAY", "STRING_ARRAY"})
+//  @Param(value = {"INT", "LONG", "STRING", "BYTES", "BIG_DECIMAL", "LONG_ARRAY", "STRING_ARRAY"})
```

Comment:
nit: Should be removed or re-instated ? 

Why are `STRING_ARRAY` and `BIGDECIMAL` removed ?

Reply:
We had the same discussion below:

> I think it is useful to have them there to have easy access to other common configurations. The current configuration (always use 10k rows and only {"INT", "LONG", "STRING", "BYTES", "LONG_ARRAY"} types is useful, but keeping the larger list of params and the other rows may be also useful in more extensive tests.